tools/c7n_mailer/c7n_mailer/sqs_queue_processor.py

```diff
@@ -16,7 +16,7 @@
 ===============
 
 """
-import memcache
```

Comment:
move all the caching logic directly into the LdapLookup class, since that's the only class that uses the logic it probably belongs there